soc/arm/nordic_nrf/nrf51/power.c

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2018 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#include <zephyr.h>
+#include <nrf_power.h>
+
+#define LOG_LEVEL CONFIG_SOC_LOG_LEVEL
+#include <logging/log.h>
+LOG_MODULE_DECLARE(soc);
```

Comment:
Do we need logging here? this file doesn't seem to be using it?

Reply:
Not really.